_posts/zh/newsletters/2019-05-29-newsletter.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Bitcoin Optech Newsletter #48'
+permalink: /zh/newsletters/2019/05/29/
+name: 2019-05-29-newsletter-zh
+slug: 2019-05-29-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+---
+本周的 Newsletter 描述了新提议的 `OP_CHECKOUTPUTSHASHVERIFY` 操作码，涵盖了对 Taproot 的持续讨论，并链接到关于处理比特币交易费增加的视频演示。此外，我们的常规栏目还包括 Bech32 发送支持、最高投票的 Bitcoin Stack Exchange 问答以及流行比特币基础设施项目的显著变化。
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+{% include specials/2019-exec-briefing/fees.md %}
+
+## 行动项
+
+*本周无。*
+
+## 新闻
+
+- **<!--proposed-new-opcode-for-transaction-output-commitments-->****提议的新交易输出承诺操作码：** Jeremy Rubin 在 Bitcoin-Dev 邮件列表中发布了一项提议，建议通过软分叉引入 `OP_CHECKOUTPUTSHASHVERIFY` 操作码，该操作码允许比特币地址要求花费该地址的交易包含一组特定的输出。这使得比特币契约的一种受限制形式成为可能，可以在某些情况下减少需要上链的数据量，从而可能降低成本或改善隐私。详情请参见本 Newsletter 的[提案特别部分][special section about the proposal]。
+
+- **<!--continued-discussion-about-bip-taproot-and-bip-tapscript-->****关于 bip-taproot 和 bip-tapscript 的持续讨论：** 本周 Bitcoin-Dev 邮件列表中的两条评论显得特别值得注意：
+
+  - **<!--final-stack-empty-->***最终堆栈为空：* 在传统、segwit 和提议的 [bip-tapscript][] 脚本中，如果脚本仅包含一个*真*值元素，则脚本评估成功。Russell O'Connor [提出][emptystack1]了他之前[提出过][emptystack0]的一个观点，并请求利用这个机会要求 tapscript 仅在以空堆栈结束时才评估成功。Pieter Wuille [回复][emptystack reply]说，他在 [miniscript][] 上的工作（参见 [Newsletter #32][]）表明，对于 miniscript 将创建的脚本子集，这种语义变化最多将节省 0.25 vbytes。此外，虽然这种变化可能简化手工编写脚本的开发，但它有一定的风险，因为迄今为止所有的 Script 开发指南都教导脚本必须在堆栈上终止时有一个*真*值。Wuille 总结道：“所以总体来说，这看起来像是有边际成本，但也最多只有边际收益的事情。”
+
+  - **<!--move-the-oddness-byte-->***移动奇数字节：* 比特币公钥最自然的表示方式是使用 X、Y 坐标对，就像比特币早期使用[未压缩公钥][uncompressed public keys]一样。然而，由于有效的公钥必须位于椭圆曲线上，因此可以根据曲线公式为任何给定的 X 坐标找到两个有效的 Y 坐标（一个奇数，一个偶数）。在压缩密钥格式中，第一个字节包含一个位，用于指定 Y 坐标是奇数还是偶数，然后是 32 字节用于编码 X 坐标。提议的 bip-taproot 遵循这一惯例，使用 33 字节编码 taproot 输出密钥。
+
+    本周，John Newbery [建议][smaller v1 spk]我们使用某种方法避免在 scriptPubKey 中放置此字节。Wuille 同意这可能有用，并将尝试实现一种变体，其中该位将作为 taproot 见证数据的一部分。这将减少创建一个 taproot 输出的成本一个 vbyte（使其与当前的 P2WSH 相同）。
+
+- **<!--presentation-a-return-to-fees-->****演讲：重返费用：** 本月早些时候在纽约区块链周期间，Bitcoin Optech 贡献者 Mike Schmidt 在 Optech 的首次高管简报会上做了关于比特币交易费用的演讲。[他演讲的视频][vid return to fees]现已上线。{{return-to-fees}}
```

Comment:
The {{return-to-fees}} here references the include for specials/2019-exec-briefing/fees.md above.

I think we will want to have a specials/2019-exec-briefing/zh/fees.md translated to zh and included instead of the /en/ version

Reply:
Let me try!